File: CLASS2/2839.py
import sys
input = sys.stdin.readline

n = int(input())
w =0
while n>=0:
    if n % 5 == 0:
        w += n // 5
        break
    else:
        w += 1
        n -= 3

if n < 0:
    print(-1)
else:
    print(w)

File: solved.ac/CLASS2_JS/2839.js
const fs = require("fs");
const inputPath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const stdin = fs
  .readFileSync(inputPath)
  .toString()
  .split("\n")
  .map((line) => line.trim());

const input = (() => {
  let line = 0;
  return () => stdin[line++];
})();

const solution = () => {
  let n = Number(input());
  let w = 0;

  while (n >= 0) {
    if (n % 5 === 0) {
      w += parseInt(n / 5);
      break;
    } else {
      w += 1;
      n -= 3;
    }
  }
  if (n < 0) {
    console.log(-1);
  } else {
    console.log(w);
  }
};

solution();
